### ISSUE TITLE
Help - Adding Oracle support

### ISSUE THREAD

**Issue by levouh** [joe-re/sql-language-server#106]
I have beyond zero experience with JS-flavor languages, but am interested in adding _Oracle_ support. Would the best place to base my ventures off of be something like [this](https://github.com/joe-re/sql-language-server/commit/6d9fdff59a43727a236b5e171d8075bc0fc57d78) commit (which added _BigQuery_)?

Any other general advice in this respect?

**Comment by joe-re** [comment]
@levouh Hi, thanks for your kind offer! Sure, support additional vendor is always welcome!
If you need some our help, please feel free to ping me, thanks. 🙏

**Comment by levouh** [comment]
Well so the first question was generally around what a good commit/_PR_ to base my changes off of would be. Do you recommend the one that [added BigQuery](https://github.com/joe-re/sql-language-server/commit/6d9fdff59a43727a236b5e171d8075bc0fc57d78) or something else @joe-re?

**Comment by joe-re** [comment]
@levouh Yes, that PR would be a good example. Basically getting DB info is going to be vendor-specific implementation.
`database_libs` (packages/server/src/database_libs) is expected to wrap this specific implementation part.
So if we want to support additional vendor(in this case, for Oracle) adding `database_libs/OracleClient.ts` would be the first step.
